apps/aecore/src/aec_conductor.erl

```diff
@@ -711,12 +723,14 @@ create_block_candidate(#state{keys_ready = false} = State) ->
     %% Keys are needed for creating a candidate
     wait_for_keys(State);
 create_block_candidate(State) ->
```

Comment:
I would very much prefer to solve this in a way that we didn't need to copy the full state to the worker process, but this is out of scope for this ticket.